### ISSUE TITLE
Cannot set an aria-label to the datatable expand row button

### ISSUE THREAD

**Issue by sylven32** [primefaces/primereact#1682]
Not found a prop to do that, see https://github.com/primefaces/primereact/blob/c2514615430ee52f2fe05ef4b4432fa5ad3b18fa/src/components/datatable/RowTogglerButton.js#L40

**Comment by mertsincan** [comment]
Hi,

Fixed in https://github.com/primefaces/primereact/issues/1762

You can create a custom expanded button using body method's parameters.

Best Regards,
